src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php

```diff
@@ -108,6 +110,11 @@ private function getFile($id, $mkdir = false)
         return $dir.substr($hash, 2, 20);
     }
 
+    private function getTime()
+    {
+        return $this->deterministicTime !== null ? $this->deterministicTime : time();
```

Comment:
Thanks @lstrojny for this PR. I think this is going too far :)
This completely breaks the expiration mechanism of cache pools.
It cannot pass PSR-6 when this flag is on anymore.
Instead, I'd suggest to use a time-less value for non-expirable items.

Reply:
On breaking expiration mechanism: doesn't that depend on what I configure as a timestamp. Since I am touching the clock, I might invalidate it too early or too late, but that's my choice as a user.
On PSR-6: can you elaborate where the PSR-6 breakage comes from?